### ISSUE TITLE
panic on unwrap on a None value 

### ISSUE THREAD

**Issue by blankenshipz** [jrmuizel/pdf-extract#66]
I'm getting a strange panic when reading an arbitrary PDF file

```
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', /Users/me/.cargo/registry/src/index.crates.io-6f17d22bba15001f/pdf-extract-0.6.5/src/lib.rs:955:37
```
I can't find any other issues that seem to be in this same area of the code; is the PDF malformed?

**Comment by jrmuizel** [comment]
Can you link to or attach the pdf?

**Comment by scotow** [comment]
I'm getting the same issue but unfortunately I can't share the PDF that I'm using for privacy reason. @jrmuizel, I reached you over Discord.

**Comment by jrmuizel** [comment]
@scotow I've fixed your PDF in deb3aea57ab8dd75212236d5b191763d0f6d4e10

**Comment by scotow** [comment]
Thanks. Seems to be working.
Do you plan to publish a new version to crates.io with this fix anytime soon?

**Comment by jrmuizel** [comment]
I published 0.7.1
